package.json

```diff
@@ -26,6 +26,7 @@
     "@ensdomains/content-hash": "^2.5.3",
     "@ethersproject/abi": "^5.0.1",
     "@ethersproject/address": "^5.0.1",
+    "@ethersproject/bignumber": "^5.2.0",
```

Comment:
I don't understand why this didn't already exist? We are importing from `@ethersproject/bignumber` already in multiple places...

Reply:
I saw some instances of BigNumber coming from the SOR but if I used those then I had errors coming up from presumably clashing versions.